src/ArborX_LinearBVH.hpp

```diff
@@ -268,13 +280,18 @@ BoundingVolumeHierarchy<MemorySpace, Enable>::BoundingVolumeHierarchy(
       space, primitives, permutation_indices, morton_indices, getLeafNodes(),
       getInternalNodes());
 
+  Kokkos::deep_copy(
+      space, Kokkos::View<BoundingVolume, Kokkos::HostSpace>(&_bounds),
+      Kokkos::View<BoundingVolume, MemorySpace, Kokkos::MemoryUnmanaged>(
+          &getBoundingVolume(getRoot())));
```

Comment:
Comment about checking for potential performance regression still holds here

Reply:
Looked ok to me.  Here is a link to the last build (running at the moment I post this)
https://code.ornl.gov/ecpcitest/alexa/-/jobs/901166